modules/nw-egress-router-pod-ovn.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc
+
+ifeval::["{context}" == "deploying-egress-router-cni-redirection"]
+:redirect:
+:router-type: redirect
+:image-name: FIXME
```

Comment:
This might vary between OCP and OKD, as well.

Reply:
@danielmellado , @weliang1 , I'm too confused to even hazard a guess.  Will Multus in OKD also copy the binary?  If yes, do we need a "wait/sleep" image and are there separate images for OCP and OKD.

I'm really looking forward to the day when I can answer some of my own questions.

Reply:
hmmm, well, the image really gets copied by the multus' daemonset on CNO. Not sure if that really applies to OKD